src/wallet/utils/isWalletACoinbaseSmartWallet.ts

```diff
@@ -25,7 +25,7 @@ export async function isWalletACoinbaseSmartWallet({
     if (!code) {
       // no code at address, check that the initCode is deploying a Coinbase Smart Wallet
       // factory address is first 20 bytes of initCode after '0x'
-      const factoryAddress = userOp.initCode.slice(0, 42) as Address;
+      const factoryAddress = userOp?.initCode?.slice(0, 42) as Address;
```

Comment:
can update [test](https://github.com/coinbase/onchainkit/blob/main/src/wallet/utils/isWalletACoinbaseSmartWallet.test.ts#L19) and [docs](https://github.com/coinbase/onchainkit/blob/main/site/docs/pages/wallet/is-wallet-a-coinbase-smart-wallet.mdx) and I think you should be able to remove permissionless dependency